src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3604,11 +3604,11 @@ components:
             $ref: '#/components/schemas/UniqueIdProperty'
       description: >
         A specification of a table, columns, and row ids to go into a snapshot.
-    SnapshotRequestAccessIncludeModel:
+    SnapshotRetrieveIncludeModel:
```

Comment:
That's fine but this is a breaking change for folks using the latest client so you'll want to add a note on the dsp-jade channel